tls/rsa.c

```diff
@@ -148,10 +150,16 @@ __rsa_setup_ctx(TlsRSACtx *ctx)
 		} while (ttls_mpi_cmp_int(vi, 1));
 		/* Blinding value: Vi =  Vf^(-e) mod N */
 		ttls_mpi_inv_mod(vi, vf, &ctx->N);
-		ttls_mpi_exp_mod(vi, vi, &ctx->E, &ctx->N, &ctx->RN);
+		if ((r = ttls_mpi_exp_mod(vi, vi, &ctx->E, &ctx->N, &ctx->RN)))
```

Comment:
For many similar places a `TTLS_MPI_CHK()` macro is used. Is it going to be deprecated or it was just lost here?

Reply:
Franky, I've never liked the macros and this is mostly why they use WARN statements. Most of the errors from mbed TLS are about failed memory allocations, but we should not do any memory allocations in fast path, so the code is going to have more void-returning calls and there won't be so many calls checking the status. In general, the macros are needed to handle many calls returning some error codes and the conditions aren't good for performance. So I'd prefer to feel pain writing the code with many barnches.